include/screencast_common.h

```diff
@@ -179,6 +179,7 @@ struct xdpw_wlr_output {
 	int width;
 	int height;
 	float framerate;
+	int32_t transformation;
```

Comment:
Using the `enum wl_output_transform` instead of `int32_t` would make it clearer what the values are.

Reply:
Done